backend/src/bundles/users/types/user-update-response-dto.ts

```diff
@@ -0,0 +1,14 @@
+import { type Gender } from './types.js';
+
+type UserUpdateResponseDto = {
```

Comment:
Partial

Reply:
fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/98/commits/bcf52b7e3b0dca9d81bd78c4ca1be95523bc8c28](url)